.gitmodules

```diff
@@ -28,3 +28,6 @@
 [submodule "libraries/libfc/libraries/ff"]
 	path = libraries/libfc/libraries/ff
 	url = https://github.com/AntelopeIO/libff
+[submodule "libraries/libfc/libraries/bn256"]
+	path = libraries/libfc/libraries/bn256
+	url = https://github.com/AntelopeIO/bn256.git
```

Comment:
All of the other submodules are without `.git` at the end. It doesn't affect anything, but for consistency consider making it like the others.

Reply:
resolve with https://github.com/AntelopeIO/leap/pull/280/commits/1404bb83e7c81bc50a2d4b0c7ba70b787870badc